src/main/java/org/havenapp/main/ui/CameraViewHolder.java

```diff
@@ -142,7 +140,7 @@ public CameraViewHolder(Activity context, CameraView cameraView) {
                 listener.onProcess(detectedImage,rawBitmap,motionDetected);
 
             if (motionDetected)
-                mEncodeVideoThreadPool.execute(() -> saveDetectedImage(rawBitmap));
+                mEncodeThreadPool.execute(() -> saveDetectedImage(rawBitmap));
```

Comment:
explanation: video encoding is done by CameraView now, so I renamed mEncodeVideoThreadPool to not mention video

Reply:
Generic var names are okay but in this case it’s going to confuse those like @archie94 who put a lot of work into it when they return to developing the project

Reply:
I'm sorry I never addressed these line replies.  Somehow I never saw them.  I definitely don't want to make it harder for returning developers.  I finally saw these in the email regarding closing of the pull request because the other had been merged.

Reply:
No worries. Happy for another pull request round for clarity sake. The good news is that video capture is working better than ever!